_posts/workshops/python-open-science-tool-setup/2018-07-20-setup-03-how-to-access-and-use-shell-to-setup-a-working-directory.md

```diff
@@ -3,8 +3,8 @@ layout: single
 category: courses
 title: 'How to Access and Use Shell to Set Up a Working Directory'
 excerpt: 'This tutorial walks you through how access the shell through terminal, use basic commands in the terminal for file organization, and set up a working directory for the course.'
-authors: ['Martha Morrissey','Leah Wasser', 'Data Carpentry']
-modified: 2018-08-31
+authors: ['Leah Wasser', 'Martha Morrissey']
```

Comment:
we can merge this but essentially you should be on this lesson too. i don't have to be first author on everything.